src/main/java/com/solvd/webtest/components/ResultBase.java

```diff
@@ -0,0 +1,25 @@
+package com.solvd.webtest.components;
+
+import com.zebrunner.carina.webdriver.gui.AbstractUIObject;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+
+public abstract class ResultBase extends AbstractUIObject {
+    public ResultBase(WebDriver driver, SearchContext searchContext) {
+        super(driver, searchContext);
+    }
+
+    public abstract String getItemTitle();
```

Comment:
Move all the methods that are common for every platform to the base classes

Reply:
I dont`t think I can move them because there are no elements declared in Base class